runners/spark/src/main/java/org/apache/beam/runners/spark/io/MicrobatchSource.java

```diff
@@ -70,13 +64,13 @@
   private final int splitId;
 
   MicrobatchSource(
-      UnboundedSource<T, CheckpointMarkT> source,
-      Duration maxReadTime,
-      int numInitialSplits,
-      long maxNumRecords,
-      int splitId,
-      int sourceId,
-      double readerCacheInterval) {
+      final UnboundedSource<T, CheckpointMarkT> source,
+      final Duration maxReadTime,
+      final int numInitialSplits,
+      final long maxNumRecords,
+      final int splitId,
+      final int sourceId,
+      final double readerCacheInterval) {
```

Comment:
Yeah, having experienced the joy of Scala's `val`, I use the make-every-thing-you-can-final inspection in IntelliJ.